apps/vgo-cli/src/vgo_cli/process.py

```
@@ -2,45 +2,198 @@
 
 def print_process_output(result: subprocess.CompletedProcess[str]) -> None:
+    """Forward captured subprocess output streams to the current process."""
     if result.stdout:
         sys.stdout.write(result.stdout)
     if result.stderr:
         sys.stderr.write(result.stderr)
 
 
 def run_subprocess(command: Sequence[str], *, cwd: Path | None = None) -> subprocess.CompletedProcess[str]:
+    """Run a subprocess with captured UTF-8 text output."""
     return subprocess.run(list(command), cwd=cwd, capture_output=True, text=True)
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)